db/migrate/20160414185708_create_container_label_tag_mappings.rb

```diff
@@ -0,0 +1,12 @@
+class CreateContainerLabelTagMappings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_label_tag_mappings do |t|
```

Comment:
I'm not sure this should be specific to containers, but I think I'm ok with it as a first pass, and then if we try to implement this for say, amazon, we can always rename it.